psi4/src/psi4/libmints/rel_potential.cc

```diff
@@ -31,71 +31,55 @@
 #include "psi4/libmints/integral.h"
 #include "psi4/libmints/matrix.h"
 #include "psi4/libmints/molecule.h"
+#include "psi4/libpsi4util/PsiOutStream.h"
 
-#include <libint2/shell.h>
 #include <libint2/engine.h>
 
-#include <algorithm>
-
 #define MAX(a, b) ((a) > (b) ? (a) : (b))
 
 #define RELVDEBUG 0
 
 ;
 using namespace psi;
 
-// Initialize potential_recur_ to +1 basis set angular momentum
 RelPotentialInt::RelPotentialInt(std::vector<SphericalTransform>& st, std::shared_ptr<BasisSet> bs1,
                                  std::shared_ptr<BasisSet> bs2, int deriv)
     : OneBodyAOInt(st, bs1, bs2, deriv) {
     if (deriv > 0) {
         throw PSIEXCEPTION("RelPotentialInt: deriv > 0 is not supported.");
```

Comment:
Just for curiosity: will be there be native support for derivatives with this and near-future set of changes? I am asking because one of the reasons to implement the integrals natively in libint2 was to make this possible.

Reply:
It's already possible in L2. I'd just need to double-check that the L2 version Psi uses has those derivatives, and then double-check how the derivatives interface with the fact that this is a multi-component operator.

Reply:
Ah. Yes, this PR _requires_ 2.8.0. How do I enforce that dependency?

Reply:
* update https://github.com/psi4/psi4/blob/master/external/upstream/libint2/CMakeLists.txt#L38
* comment out COMPONENTS and OPTIONAL_COMPONENTS below it
* same here https://github.com/psi4/psi4/blob/master/psi4/CMakeLists.txt#L180-L191

Reply:
Does that mean commenting out the parts in all caps, or also the lines under them?

Reply:
lines under them, too. upstream L2 doesn't export any components.